setup.py

```diff
@@ -24,15 +24,15 @@
         'numpy',
         'scikit-learn',
         'matplotlib',
-        'colour',
-        'pytest'
+        'colour'
     ],
     extras_require={
         'xgboost': extra_xgboost,
         'pyspark': extra_pyspark,
         'lightgbm': extra_lightgbm,
         'tensorflow_decision_forests': extra_tensorflow,
         'all': extra_xgboost + extra_pyspark + extra_lightgbm + extra_tensorflow,
+        'develop': ['pytest']
```

Comment:
could we add "develop" (or "dev" in case we are gonna change it) into "all" also ?

Reply:
I don't think that we should. I think that all current extras, including `all`, are production extras (required at runtime to use the library). The future extra `dev`/`develop`/`test` has a different purpose which doesn't really fit the existing extras.

It's always possible to install both with `pip install dtreeviz[all,dev]` for instance.

Reply:
Agree! :)